### ISSUE TITLE
Licence of modifided OFL font subset used as vector image (SVG)

### ISSUE THREAD

**Issue by tsinis** [flutter/flutter_clock#41]
Hey Google!

To begin with, I want to thank you for the great competition that I can be a part of! So, I have downloaded a font from fonts.google.com (which is licensed as OFL), but:

1. i'm using only numbers from that font,
2. I have slightly changed the way it look
3. and the most important thing — **I'm using it as vector image** (SVG).

So question is — **do I need to specify the original font and it's license?** Thanks in advance!

Best regards,
Roman

**Comment by JonathanMonga** [comment]
Yes, it is recommended... you must specify the license of all assets in your project and OFL license for all fonts
